app/controllers/styleguide_controller.rb

```diff
@@ -16,7 +16,8 @@ def components
       'Latest news' => 'latestnews',
       'Download box' => 'downloads',
       'Feedback' => 'feedback',
-      'Large call to action' => 'largecta'
+      'Large call to action' => 'largecta',
+      'Side panel' => 'sidepanel'
```

Comment:
This has been moved to it's own helper. It's nothing you need to deal with just letting you know.